profiling-ffi/src/exporter.rs

```diff
@@ -121,15 +121,15 @@ pub extern "C" fn ddog_prof_Exporter_new(
     profiling_library_name: CharSlice,
     profiling_library_version: CharSlice,
     family: CharSlice,
-    tags: Option<&ddcommon_ffi::Vec<Tag>>,
+    tags: *const ddcommon_ffi::Vec<Tag>,
     endpoint: Endpoint,
 ) -> ExporterNewResult {
     match || -> anyhow::Result<ProfileExporter> {
         let library_name = unsafe { profiling_library_name.to_utf8_lossy() }.into_owned();
         let library_version = unsafe { profiling_library_version.to_utf8_lossy() }.into_owned();
         let family = unsafe { family.to_utf8_lossy() }.into_owned();
         let converted_endpoint = unsafe { try_to_endpoint(endpoint)? };
-        let tags = tags.map(|tags| tags.iter().map(Tag::clone).collect());
+        let tags = unsafe { tags.as_ref() }.map(|tags| tags.iter().map(Tag::clone).collect());
```

Comment:
Just to doublecheck -- is it still ok to pass in a `NULL` for the tags?

Reply:
Yes.